plugins/inputs/mavlink/sample.conf

```diff
@@ -0,0 +1,37 @@
+# Read metrics from a Mavlink connection to a flight controller.
+[[inputs.mavlink]]
+     ## Flight controller URL. Must be a valid Mavlink connection string in one
+     ## of the following formats:
+     ##
+     ## - Serial port:  serial://<device name>:<baud rate> 
+     ##            eg: "serial:///dev/ttyACM0:57600"
+     ## 
+     ## - TCP client:   tcp://<target ip or hostname>:<port>
+     ##            eg: "tcp://192.168.1.12:14550"
+     ## 
+     ## - UDP client:   udp://<target ip or hostname>:<port>
+     ##            eg: "udp://192.168.1.12:14550"
+     ## 
+     ## - UDP server:   udp://:<listen port>
+     ##            eg: "udp://:14540"
+     ## 
+     ## The meaning of each of these modes is documented by
+     ## https://mavsdk.mavlink.io/v1.4/en/cpp/guide/connections.html.
+     fcu_url = "udp://:14540"
+
+     ## Filter to specific messages. Only the messages in this list will be parsed.
+     ## If blank or unset, all messages will be accepted.
+     # message_filter = ["global_position_int", "attitude"]
```

Comment:
Commented out paramaters should contain the default value. This seems not to be the case here?